### PR TITLE
Actuals and Budget in projects.list/info and milestones.list/info

### PR DESCRIPTION
**Pull request by adognameddog**

### CODE REVIEW

**Review by mikeverf — DISMISSED** [review]

File: src/07-projects/milestones.apib [hunk 1]
```diff
@@ -104,6 +117,19 @@ Get details for a single milestone.
                 + (object)
                     + type: `milestone` (string)
                     + id: `b71ff11f-074e-0705-9450-589342af4f4f` (string)
+            + actuals (object) - Only accessible for administrators of the project this milestone belongs to
+                + `billable_amount` (Money)
+                + costs (Money)
+                + result (Money) - The billable amount minus the costs, so the profit or loss
+            + budget (object) - Only accessible for administrators of the project this milestone belongs to
+                + provided (Money)
+                + spent (object)
```
Comment (ketlane) [inline src/07-projects/milestones.apib]: Spent is exposed as `Money` object on `milestones.info`

File: src/07-projects/milestones.apib [hunk 2]
```diff
@@ -104,6 +117,19 @@ Get details for a single milestone.
...
+                    + total (Money)
+                    + time (Money)
+                    + materials (Money)
+                + remaining (Money)
+                + allocated (Money, nullable) - The amount of money still expected to be spent. This is null if there is not enough data to calculate
+                + forecasted (Money, nullable) - A prediction of the spent budget upon milestone completion. This is null if there is not enough data to calculate
```
Comment (ketlane) [inline src/07-projects/milestones.apib:127]: On milestones, we have `projected` budget instead of `forecasted` budget

**Comment by adognameddog** [comment]
We need to refactor our frontend app, so it uses "forecasted" instead of "projected". This PR is put on hold

**Comment by ketlane** [comment]
The ticket for refactoring: https://teamleader.atlassian.net/browse/PRO-2323

File: src/07-projects/milestones.apib (hunk 1 above)
Comment (adognameddog) [reply src/07-projects/milestones.apib]: Yes, but here it's divided into total, time, and materials. So a regular object

File: src/07-projects/milestones.apib (hunk 2 above)
Comment (adognameddog) [reply src/07-projects/milestones.apib:127]: It should be forecasted. We are changing the implementation on the front and back end

**Review by ketlane — APPROVED** [review]

**Comment by ketlane** [comment]
:white_check_mark: Manually checked

**Review by mikeverf — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**